tests/test_absolutize.py

```diff
@@ -55,8 +54,9 @@ def test_make_paths_absolute(self, workflow_id):
             assert subjects[0] != subjects[1]
 
     def test_merged(self):
+        manifiest_file = get_test_data_file("manifest.json")
```

Comment:
Can we use `manifest_file` not `manifiest_file` here, or will that clash with another variable name?